src/app/invites/invite.services.ts

```diff
@@ -0,0 +1,25 @@
+// invites/invite.service.ts
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+export interface Invite {
+  id: number;
+  when: string;
+  status: string;
+  who: { name: string };
+  what: { title: string };
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class InviteService {
+  private apiUrl = 'http://localhost:8080/api/invites';
```

Comment:
You shouldn't fix the backend URL because it will not work when deployed on the cloud. If you implement the service from scratch using HttpClient (instead of using the HateoasClient), follow the same approach as in https://github.com/UdL-EPS-SoftArch/tfgfinder-gei/blob/main/src/app/interest/interest.service.ts